Resources/Prototypes/Roles/Jobs/departments.yml

```diff
@@ -79,6 +79,7 @@
   - Psychologist
   - Paramedic
   - MedicalBorg  # Delta V - Medical Borg, see Resources/Prototypes/DeltaV/Roles/Jobs/Medical/medicalborg.yml
+  - Pharmacist # Delta V - Pharmacist, see Ressources/Prototypes/DeltaV/Roles/Jobs/Medical/pharmacist.yml
```

Comment:
After discussion with Velcro, it sounds like this role will be used as a station-specific role, so it should be marked accordingly.